src/CoreBundle/TemplateEngine/Mapper/ModuleChooserModuleListMapper.php

```diff
@@ -76,7 +75,7 @@ public function Accept(
                 );
             }
             usort($module['views'], function ($element1, $element2) {
-                return $element1['displayName'] > $element2['displayName'];
+                return $element1['displayName'] > $element2['displayName'] ? 1 : 0;
```

Comment:
Sorting methods should return integers but this one returned a bool before. Casting a bool to int yields `1` or `0`, so this should not change the behaviour but now uses correct types